Sources/UIImage+Sweet.swift

```diff
@@ -3,6 +3,26 @@
 
     public extension UIImage {
 
+        /// Creates a QR code from a string.
+        ///
+        /// - Parameter string: Text to be the QR Code content
+        /// - Returns: image QR Code image
+        public static func imageQRCode(for string: String) -> UIImage {
```

Comment:
Can you add unit test for this? At least verifying that calling this method with two different strings output different UIImages.